app/src/main/java/io/github/mattpvaughn/chronicle/features/currentlyplaying/CurrentlyPlayingFragment.kt

```diff
@@ -125,17 +123,14 @@ class CurrentlyPlayingFragment : Fragment() {
             currentlyPlayingInterface.setBottomSheetState(COLLAPSED)
         }
 
-        viewModel.showModalBottomSheetSpeedChooser.observe(viewLifecycleOwner) { visible ->
-            if(visible) modalBottomSheetSpeedChooser.show(childFragmentManager, ModalBottomSheetSpeedChooser.TAG)
-        }
-
-        viewModel.speed.observe(viewLifecycleOwner) { value ->
-            Timber.i("viewModel.speed.observe → $value")
-            // todo: remove → only for debugging
-        }
-        viewModel.playbackSpeedString.observe(viewLifecycleOwner) { value ->
-            Timber.i("viewModel.playbackSpeedString.observe → $value")
-            // todo: remove → only for debugging
+        viewModel.showModalBottomSheetSpeedChooser.observe(viewLifecycleOwner) { eventShowChooser ->
+            if (!eventShowChooser.hasBeenHandled) {
```

Comment:
emitting this as an event since the `ModalBottomSheetSpeedChooser` is a fragment and survives config changes. event is cleared by `getContentIfNotHandled`

Minor issue, but previously, if you rotated the screen over and over (causing `CurrentlyPlayingFragment` to get recreated over and over and keep observing this `showModalBottomSheetChooser` over and over), a bunch of choosers would stack up on top of each other.